appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -45,7 +45,7 @@ export async function getDeployFsPath(context: IActionContext, target: vscode.Ur
         }
 
         const selectFile: string = localize('selectDeployFile', 'Select the {0} file to deploy', fileExtensions ? fileExtensions.join('/') : '');
-        const selectFolder: string = localize('selectZipDeployFolder', 'Select the folder to zip and deploy');
+        const selectFolder: string = trialApp ? localize('selectDeployFolder', 'Select the folder to deploy') : localize('selectZipDeployFolder', 'Select the folder to zip and deploy');
```

Comment:
Just use "Select the folder to deploy" in all cases. Not worth the complexity to make them different